ontology/uco/core/core.ttl

```diff
@@ -518,6 +556,27 @@ core:endTime
 	rdfs:range xsd:dateTime ;
 	.
 
+core:eventAttribute
+	a owl:ObjectProperty ;
+	rdfs:label "Event Attribute"@en ;
+	rdfs:comment "An event attribute specifies an ad-hoc attribute/value for an event."@en ;
+	rdfs:range types:Dictionary ;
```

Comment:
There's an OWL Issue with this `rdfs:range`: `core:` does not import `types:`.

We can resolve this by either:
* Importing `types:` (which I doubt we want to do, due to circular import issues - `types:` already imports `core:`)
* Defining a stub reference to `types:Dictionary`, by just including `types:Dictionary a owl:Class .` in `core.ttl`, and maybe attaching a `rdfs:seeAlso` to the types namespace.
* Moving `Event` to another namespace file that imports `types.ttl`.  Editorializing: Letting this have its own namespace feels a bit overkill to me.

Reply:
Another option is having `Event` be in the `time:` namespace.

@sbarnum , do you have a feel for the most preferable resolution here?

Reply:
Thank you for catching this.

**I think I would prefer to go with the stub approach in core.**
I agree with the desire to avoid the circular import issue. Giving Event its own namespace seems the cleanest approach but I agree it seems a bit of overkill to give Event its own namespace. I would prefer to avoid putting Event in the time namespace as I believe while temporal in nature, Event is more than just time and putting it in time is likely to cause confusion on this issue.

Reply:
Stub added in [9cf555a](https://github.com/ucoProject/UCO/pull/542/commits/9cf555adabcdf9ee0fba34d48059ffb017a3e42d).